modules/manage/pages/raft-group-reconfiguration.adoc

```diff
@@ -0,0 +1,66 @@
+= Raft Group Reconfiguration in Redpanda
+:description: Learn how the Redpanda Raft group protocol provides consistency and availability during reconfiguration.
+
+Redpanda and Kafka with Raft both use the Raft consensus algorithm to manage cluster metadata. This metadata is stored as messages in an internal topic in a separate Raft cluster instance, called a Raft group. The messages in this topic contain the current set of brokers that are members of a Raft group. In a Raft group, one member is the leader and the others are followers. 
+
+Additionally, Redpanda uses Raft as its replication protocol for the actual data (that is, the application data). Each topic partition also forms a Raft group comprised of its leader as well as follower brokers. See xref:get-started:architecture.adoc#raft-consensus-algorithm[How Redpanda Works] for a further explanation of how Raft manages data replication in Redpanda.
+
+== About Raft group reconfiguration
+
+A Raft group’s member set changes during operations like cluster resize or xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing]. This change, also referred to as Raft group reconfiguration, includes adding, removing, or replacing Raft group members. When the group configuration changes, the current leader replicates a new configuration message and sends it to the followers. 
+
+Raft uses the joint consensus approach to maintain consistency during reconfiguration. It involves both the old member set and the proposed new member set electing a new group leader and agreeing on the latest committed offset. The new leader can be elected from either the old or new member set. Partitions are unavailable to accept writes until the group reaches joint consensus.
```

Comment:
the last part i.e. group reaches joint consensus, i would rephrase as joint consensus is a name of reconfiguration algorithm. Here both old and new replica set must agree to accept a write or vote for a leader